crates/goose/src/prompts/system.md

```diff
@@ -10,6 +10,14 @@ to interactions with sytems that are not currently active. The currently
 active systems are below. Each of these systems provides tools that are
 in your tool specification.
 
+The format of a tool is "{system_name}__{tool_name}", i.e. the system
```

Comment:
nit: i think these additions are excludable? not sure if you saw any performance issues with this but i have not seen any problems or a need to warn it about tools beyond the tool descriptions